qa/tasks/cram.py

```diff
@@ -33,6 +34,7 @@ def task(ctx, config):
               - qa/test2.t]
               client.1: [qa/test.t]
             branch: foo
+            noparallel: True
```

Comment:
```suggestion
            parallel: False
```

and `parallel` is `True` by default.

Reply:
Done, thanks.